packages/@coorpacademy-components/src/molecule/modal/index.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable react/no-danger */
+/* eslint-disable react/jsx-no-comment-textnodes */
+
+import React from 'react';
+import {
+  NovaLineContentEditionQuillCircle as QuillCircle,
+  NovaSolidRemoveAddAddCircle2 as AddCircle,
+  NovaSolidStatusClose as Close,
+  NovaSolidStatusCheckCircle2 as Check,
+  NovaCompositionCoorpacademyImportIcon as ImportIcon
+} from '@coorpacademy/nova-icons';
+import PropTypes from 'prop-types';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const Choice = ({title, subtitle, link, children}) => {
+  return (
+    <a className={style.choice} href={link}>
+      <div className={style.choiceContent}>
+        <div className={style.choiceLeftIconWrapper}>{children}</div>
+        <div className={style.choiceDescription}>
+          <h3 className={style.choiceDescriptionTitle}>{title}</h3>
+          <span className={style.choiceDescriptionSubtitle}>{subtitle}</span>
+        </div>
+      </div>
+      <div className={style.choiceRightIconWrapper}>
+        <Check className={style.choiceRightIcon} />
+      </div>
+    </a>
+  );
+};
+
+const Modal = props => {
+  const {header, description, choices, onClose} = props;
+  return (
+    <div className={style.modalWrapper}>
+      <div className={style.modal}>
+        <header className={style.header}>
+          <div className={style.headerRightContent}>
+            <div className={style.headerRightIconWrapper}>
+              <AddCircle className={style.headerAddIcon} />
+              <div className={style.shadow} />
+            </div>
+            <h3>{header}</h3>
+          </div>
+          <Close onClick={onClose} className={style.headerCloseIcon} />
+        </header>
+        <div className={style.description} dangerouslySetInnerHTML={{__html: description}} />
```

Comment:
`dangerouslySetInnerHTML` ? are you sure about this ?

Reply:
normalement cette lib est configurée pour autoriser que certaines balises